shard.yml

```diff
@@ -1,9 +1,9 @@
 name: future
-version: 0.1.0
+version: 1.0.0
 
 authors:
   - Brian J. Cardiff <bcardiff@manas.tech>
 
-crystal: 0.34.0
+crystal: ">= 0.34.0, < 2.0.0"
```

Comment:
why the `< 2.0.0` part is needed here?

Reply:
in case of breaking changes in crystal 2.0.0?
happy to remove it

Reply:
I would remove it for now. Since there is no understanding if the library is going to be incompatible with 2.0